src/editor/Core/Types.re

```diff
@@ -32,15 +32,21 @@ module Mode = {
   type t =
     | Insert
     | Normal
+    | Replace
+    | Visual
+    | Operator
     | Commandline
     | Other;
 
   let show = v =>
```

Comment:
@CrossR just a heads up #83 my change in #76 will actually remove this function as we're going to use a package which adds a show function for types, not much to worry about here but if my change is merged first it will conflict with this

Reply:
to clarify only the show function will be removed but the package will still return a string for each type based on what they are called in the type definition so will appear the same